File: add.py
import sys

try:
  first = int(sys.argv[1])
except (IndexError, NameError, ValueError):
  print("first input not defined or not a number")
  sys.exit(1)

try:
  second = int(sys.argv[2])
except (IndexError, NameError, ValueError):
  print("second input not defined or not a number")
  sys.exit(1)

print(first + second)

File: add.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func main() {

	if len(os.Args) == 1 {
		fmt.Println("first input is missing")
		os.Exit(0)
	} else if len(os.Args) == 2 {
		fmt.Println("second input is missing")
		os.Exit(0)
	}

	first, err1 := strconv.Atoi(os.Args[1])
	second, err2 := strconv.Atoi(os.Args[2])

	if err1 != nil {
		fmt.Println("first input is not a number.")
		os.Exit(0)
	} else if err2 != nil {
		fmt.Println("second input is not a number.")
		os.Exit(0)
	}

	fmt.Println(first + second)

}
